### DISCUSSION TITLE
如何自己修改device.json内容

### DISCUSSION THREAD

**Discussion by mentlesoul** [Mrs4s/go-cqhttp#1430] [Q&A]
如题，想知道device.json中除了protocol以外的内容在手机上对应那一部分
猜测把device.json数据修改的和常用登录手机一样可以减少被风控的概率

**Comment by Miuzarte** [comment]
自用的 你参考参考
![image](https://user-images.githubusercontent.com/66856838/159106485-b64a4852-0d54-47da-b66e-d630d5768978.png)
